node-sdk-demo/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "workspaces-demo-nodejs",
+  "version": "1.0.0",
+  "description": "Signadot workspaces JS SDK demo",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --timeout 60000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/signadot/workspaces-demo-nodejs.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/signadot/workspaces-demo-nodejs/issues"
+  },
+  "homepage": "https://github.com/signadot/workspaces-demo-nodejs#readme",
+  "devDependencies": {
+    "@babel/core": "^7.16.7",
+    "@babel/node": "^7.16.8",
+    "@babel/preset-env": "^7.16.8",
+    "chai": "^4.3.4",
+    "mocha": "^9.1.3",
+    "nodemon": "^2.0.15"
+  },
+  "type": "module",
+  "dependencies": {
+    "axios": "^0.24.0",
+    "nanoid": "^3.1.32"
```

Comment:
Didn't need the sdk because of local linking. But needs to be added before merge (while testing)